src/krux/display.py

```diff
@@ -365,6 +365,9 @@ def draw_string(self, x, y, text, color=theme.fg_color, bg_color=theme.bg_color)
             x = max(0, x)
         lcd.draw_string(x, y, text, color, bg_color)
 
+    def _get_text_lines(self, text, max_lines=None):
```

Comment:
To avoid having two functions with same purpose, couldn't `to_lines` have this at the beginning?
```
if isinstance(text, list):
    return text
```